module/modmanager/manager.go

```diff
@@ -150,6 +151,9 @@ type Manager struct {
 	// modPeerConnTracker must be updated as modules create/destroy any underlying WebRTC
 	// PeerConnections.
 	modPeerConnTracker *rdkgrpc.ModPeerConnTracker
+
+	failedModules   map[string]bool
+	muFailedModules sync.RWMutex
```

Comment:
style: mutexes are named with mu as a suffix, and usually above the thing it is guarding